CHANGELOG.md

```diff
@@ -2,6 +2,100 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [2.2.0](https://github.com/consuldemocracy/consuldemocracy/tree/2.2.0) (2024-0x-xx)
+
+[Full Changelog](https://github.com/consuldemocracy/consuldemocracy/compare/2.1.1...2.2.0)
+
+### Added
+
+- **Accessibility:** Add a link to skip to the main content [\#5396](https://github.com/consuldemocracy/consuldemocracy/pull/5396) and [\#5486](https://github.com/consuldemocracy/consuldemocracy/pull/5486)
+- **Documentation:** Import code and commit history from the docs repository [\#5544](https://github.com/consuldemocracy/consuldemocracy/pull/5544)
+- **Homepage:** Allow sorting homepage cards [\#5276](https://github.com/consuldemocracy/consuldemocracy/pull/5276)
+- **Maintenance:** Use Zeitwerk to autoload files [\#5425](https://github.com/consuldemocracy/consuldemocracy/pull/5425) and [\#5492](https://github.com/consuldemocracy/consuldemocracy/pull/5492)
```

Comment:
Should be in "Changed" instead of "Added" IMHO, since we're switching to a different autoloader :thinking:.